content/examples/docker/basic.docker-compose.yml.md

```diff
@@ -8,9 +10,14 @@ services:
       - ./config.yaml:/pomerium/config.yaml:ro
     ports:
       - 443:443
-
+    networks:
```

Comment:
Maybe we should add a comment that this is only required when the `localhost.pomerium.io` domain is used?